tests/contrib/grpc/test_grpc.py

```diff
@@ -118,7 +118,7 @@ def _test_insecure_channel(self, insecure_channel_function):
             stub.SayHello(HelloRequest(name="test"))
 
         spans = self.get_spans_with_sync_and_assert(size=2)
-        server_span, client_span = spans
+        client_span, server_span = spans
```

Comment:
For some reason all the grpc test spans are in reverse order. The parenting still seems fine so I'm not concerned. @majorgreys any idea why this might be? Are the client spans started before the server?

Reply:
yep, the client spans would start before server spans but client would finish after. should that matter with the reworked trace building here?

Reply:
ah that explains it then :+1:. I don't think it matters as the trace structure is the same. This is also pretty unique to the tests as typically the client and server spans wouldn't be in the same process for grpc.